### DISCUSSION TITLE
Homekit On/On Switch for A/C Zone control

### DISCUSSION THREAD

**Discussion by wrx5** [RavenSystem/esp-homekit-devices#1843] [General]
Hi All,

There is normal light switch on the wall (2 switches on the wall plate), which is in use to control aircon's zones. I have only 2 zones as upstairs and downstairs. I have opened it up to understand how it work. What I noticed is 240v is always active even if the switch is on or off. Basically it diverts 240v from one cable to another to control the related zone. 

Smart switches in the market are only On/Off switches. Mainly they have these wire connection points:

Live: L
L1: Load1 (L2, L3 if 2 or 3 gangs)
Neutral: N (some are without neutral)
G: Ground

I could get 4 gang switch and achieve what I want with automation as:

Zone1:
If Switch1 pressed : Turn on L1 and Turn off L2
If Switch2 pressed : Turn off L1 and Turn on L2

Zone2:
If Switch3 pressed : Turn on L3 and Turn off L4
If Switch4 pressed : Turn off L3 and Turn on L4

However, this is not the right way of doing it. I need to finish this with 2 gang switch as it is now. 

Do any of you know any device out there to achieve what I want? Needs work with HomeKit.

**Comment by xrust83** [comment]
Yesterday I tried this command and it works great, just write the command to turn on L1 in step 1 and then add the command m[[2.0]] which will turn off the second device, provided that you turn on the first one. It works like an interlock. It helped me

"1": {"r" ...,"m":[[2.0]],...}

https://github.com/RavenSystem/esp-homekit-devices/wiki/Accessory-Configuration#actions-example-2

Here is my json for sonoff dual

`{"c":{"l":13,"n":"Water-Faucet","b":[{"g":10,"t":5}]},"a":[{"0":{"r":[{"g":12}]},"1":{"r":[{"g":12,"v":1}],"m":[[2,0]]},"i": 60,"b":[{"g":10,"t":1}],"es":[{"0":{"r":[{"g":5}]},"1":{"r":[{"g":5,"v":1}],"m":[[1,0]]},"i": 60,"b":[{"g":10,"t":2}]}]}]} `
